src/runtime.rs

```diff
@@ -11,6 +10,16 @@ use colored::Colorize;
 use console::Term;
 use miette::Result;
 
+macro_rules! exception {
+    ( $fmt:literal $($tt:tt)* ) => {{
+        eprintln!(
+            concat!("exception: ", $fmt, ", exiting")
+            $($tt)*
+        );
+        std::process::exit(0xEE);
```

Comment:
What is the significance of 238 as an exit code?

Reply:
No reason, it's just unique. If used in a script it can be easily checked if the error was with the lc3 program.

Reply:
reasonable enough :+1: